_posts/2019-08-05-hhvm-4.17.0.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: "HHVM 4.17.0"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.17 is released! This release marks the end of support for 4.11;
+4.12-4.16 remain supported, as do the LTS releases 3.30 and 4.8.
+
+# Highlights
+
+**Reified generics** are now supported. See [full guide](https://docs.hhvm.com/hack/generics/reified-generics).
+
+Quick example:
+
+```
+abstract class BaseClass {
+  abstract function foo(): void;
+}
+
+function bar<reify T as BaseClass>(T $obj): void {
+  T::foo();  // T can now be used inside the function
```

Comment:
Let's:
- split this up (and remove the unused `T $obj` parameter)
- add a comment about this replacing the need for a `classname<T>` parameter